src/components/profile-view.ts

```diff
@@ -881,8 +988,27 @@ export class ProfileView extends LitElement {
           <sl-input label="Cash" name="apps.cash" value="${this.socialData.apps.cash}" class="label-on-left"></sl-input>
         
         </form>
+
+        <sl-button class="edit-button" size="small" @click="${e => this.credentialEditModal.show()}">
+          Verifiable Credential - Config
+        </sl-button>
         <sl-button slot="footer" variant="primary" @click="${ e => this.profileEditModal.hide() }">Submit</sl-button>
       </sl-dialog> 
+
+      <sl-dialog id="credential_edit_modal" class="page-dialog" label="Config Verifiable Credential" placement="start">
+        <small>Get an account from <a href="https://sophtron.com/account/register" target="_blank">Sophtron</a> if you don't have already.</small>
+        <form id="credential_form" @sl-change="${e => this.saveCredentialInfo(e)}" @submit="${e => e.preventDefault()}">
+          <br/>
+          <sl-input name="reverify" value="${this.credentialData.reverify}" label="Reverify option" help-text="Choose to start over from - 'ucw'(re-connect bank), 'vc'(Retrieve VC), 'verify'(Verify VC)"></sl-input>
+          <sl-input name="providerConfig.sophtronClientId" value="${this.credentialData.providerConfig?.sophtronClientId}" label="Sohtron ClientID" help-text=""></sl-input>
+          <sl-input name="providerConfig.sophtronClientSecret" type="password" value="${this.credentialData.providerConfig?.sophtronClientSecret}" label="Sohtron AccessKey" help-text=""></sl-input>
```

Comment:
It'll feel much better if we have a way to encrypt this before sending it to DWN.